wolox-books/src/app/screens/user/components/register/register.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, Validators, FormGroup } from '@angular/forms'
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.scss']
+})
+export class RegisterComponent implements OnInit {
+
+  formGroup: FormGroup;
+
+  data: any;
+
+  constructor(private formBuilder: FormBuilder) {
```

Comment:
if you don't mutate these objects they should be `readonly`
```suggestion
  constructor(private readonly formBuilder: FormBuilder) {
```